services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/ApplicationAttachmentFactory.kt

```diff
@@ -58,18 +58,25 @@ class ApplicationAttachmentFactory(
         return ApplicationAttachmentBuilder(entity, attachmentRepository, this)
     }
 
-    fun saveAttachment(applicationId: Long): ApplicationAttachmentEntity {
+    fun saveAttachmentToDb(applicationId: Long): ApplicationAttachmentEntity {
         val attachment = attachmentRepository.save(createEntity(applicationId = applicationId))
         contentService.save(attachment.id!!, DUMMY_DATA)
         return attachment
     }
 
+    fun saveAttachment(applicationId: Long): ApplicationAttachmentEntity {
+        val applicationAttachmentEntity = createEntity(applicationId = applicationId)
+        ApplicationAttachmentBuilder(applicationAttachmentEntity, attachmentRepository, this)
+            .withCloudContent()
+        return applicationAttachmentEntity
+    }
+
     fun saveContentToDb(attachmentId: UUID, bytes: ByteArray): ApplicationAttachmentContentEntity {
         val entity = ApplicationAttachmentContentEntity(attachmentId, bytes)
         return contentRepository.save(entity)
     }
 
-    fun saveContentToCloud(
+    fun saveContent(
```

Comment:
Renaming this method could wait until there's just one place to store the content in.

Reply:
Yes, you're right, I went too far in this task.

Reply:
You renamed `saveContentToDb` to `saveContent` instead?

Reply:
Sorry, that was also a mistake.